testgrid/tgapi/pkg/testrun/types/types.go

```diff
@@ -3,6 +3,11 @@ package types
 import "time"
 
 type TestRun struct {
-	ID        string
-	CreatedAt time.Time
+	ID           string     `json:"id"`
+	CreatedAt    time.Time  `json:"created_at"`
+	LastResponse *time.Time `json:"last_response"`
```

Comment:
is there some way to know if its complete so we can add duration?

Reply:
there's PendingRuns, which should be 0 when everything completes